src/api/MarkerBasedAPI.js

```diff
@@ -49,6 +50,7 @@ class MarkerBasedApi extends Base {
         limit: number,
         shouldFetchAll: boolean,
         requestData: Object = {},
+        headers?: StringMap = {},
```

Comment:
Is this change necessary? It doesn't look like this is used.

Reply:
I meant to remove those after removing `Accept-Language` header pass-through. Thanks!